pydicom/pixel_data_handlers/numpy_handler.py

```diff
@@ -284,10 +288,21 @@ def get_pixeldata(ds: "Dataset", read_only: bool = False) -> "np.ndarray":
             "the dataset"
         )
 
+    # Attributes required by both Floating Point Image Pixel Module Attributes
+    # and Image Pixel Description Macro Attributes
     required_elements = [
-        'BitsAllocated', 'Rows', 'Columns', 'PixelRepresentation',
+        'BitsAllocated', 'Rows', 'Columns',
         'SamplesPerPixel', 'PhotometricInterpretation'
     ]
+    if px_keyword == 'PixelData':
+        # Attributess required by Image Pixel Description Macro Attributes
+        required_elements.extend([
+            'PixelRepresentation',
+            'BitsStored',
+            'HighBit',
```

Comment:
Do we need to check for *High Bit*? It doesn't get used by the function (or by any calling functions)

Reply:
The motivation for adding it was to ensure the given data set contains all attributes potentially required to decode the pixel data. It's a type 1 attribute that shall be present in any data set containing a Pixel Data element. Please let me know if I should remove it again.

Reply:
The value can also be computed from `BitsStored`, so the attribute would probably not be needed in the future either. I'll remove it again.

Reply:
`HighBit` is a remnant of the time where overlays have been written into the high bits of the pixel data, but it is now defined to always have the value `BitsStored-1`. It is mandatory for historical reasons only and has no function, so I think it is ok to remove it.

Reply:
Removed the attribute from the list via 1201a5f27989e6133be68cddc10b9fcf3e3ab55c